lib/static_sync/storage.rb

```diff
@@ -49,6 +49,7 @@ def storage
         :region                => @config.remote['region'],
         :aws_access_key_id     => @config.remote['username'],
         :aws_secret_access_key => @config.remote['password'],
+        :aws_session_token     => @config.remote['session_token'],
```

Comment:
You mean if `@config.remote['session_token']` is not specified? In that case, the value of `: aws_session_token` will be `nil` and be ignored.